src/constants.py

```diff
@@ -11,13 +11,21 @@
 DINING_STACK_URL = 'https://raw.githubusercontent.com/cuappdev/DiningStack/master/DiningStack/'
 EATERY_DATA_PATH = './eatery-data/'
 GET_URL = 'https://services.get.cbord.com/GETServices/services/json'
+GET_LOCATIONS = {
+    'Attrium Cafe': 'Atrium Café',
```

Comment:
is this supposed to have 2 t's?

Reply:
no, that's why we change it

Reply:
and here i thought you made a typo, all good